packer/gpu/scripts/install-nvidia-docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# nvidia driver
+sudo yum install -y kernel-devel-$(uname -r) kernel-headers-$(uname -r) dkms
+
+sudo yum install -y pkgconfig libglvnd-devel
+
+DRIVER_CUDA_124="https://us.download.nvidia.com/tesla/550.54.15/NVIDIA-Linux-x86_64-550.54.15.run"
```

Comment:
maybe use a (readonly) bash variable for the version?

could you document why we pick this version? is there a version bound requirement?

Reply:
done